chats/models.py

```diff
@@ -11,12 +12,21 @@ class Chat(models.Model):
     def __str__(self):
         return f"{self.subject}  {self.participants}"
 
+
 class Message(models.Model):
     message_id = models.BigAutoField(primary_key=True)
-    chat = models.ForeignKey(Chat, on_delete=models.CASCADE, related_name="messages",null=True)
+    chat = models.ForeignKey(Chat, on_delete=models.CASCADE, related_name="messages", null=True)
     sender = models.ForeignKey(Companies, on_delete=models.CASCADE, related_name="sent_messages")
```

Comment:
As a suggestion: what if sender and recipient foreign keys was linked to UserAndCompanyRelations relation_id? We could extract both user and company through that table